rest_local.yml

```diff
@@ -0,0 +1,28 @@
+---
+- hosts: all
+  become: false
+  vars_files:
+    - vars/main_local.yml
+    - "{{ rest_private_dir }}/conf/release_ebi.yml"
+
+  vars:
+    PERL_RC: "{{ ensembl_install_dir }}/activate_ensembl"
+    LANG: C
+    SHARED_PATH: "{{ ensembl_install_dir }}"
+    PROGRESSIVE_CACTUS_DIR: "{{ SHARED_PATH }}/paths/progressiveCactusFake/"
+    install_my_own_hdf5: false
+    install_ensembl_hdf5: false
+    install_hdf5: false
+    do_ensembl_hdf5: false
```

Comment:
You're still preventing HDF5 installs?

Reply:
Yes. If you really want HDF5 on your local install, you should know what you're doing. At least as a default for this situation, which is people in the hallway setting up a dev environment.

Reply:
On second though, do_ensembl_hdf5 should be removed I think. This is controlled via install_ensembl_hdf5. Don't make then double turn it on if they want it.

Reply:
removed do_ensembl_hdf5. Thanks.